schemas/com.snowplowanalytics.snowplow/flash_context/jsonschema/1-0-0

```diff
@@ -36,8 +37,17 @@
 		},
 		"hasScriptAccess": {
 			"type": "boolean"
+		},
+		"domainSessionIndex": {
+			"type": "number"
+		},
+		"domainUserId": {
+			"type": "string"
+		},
+		"userFingerprint": {
+			"type": "number"
```

Comment:
Should this be a number? In `atomic.events` we have `user_fingerprint varchar(50) encode runlength,`